### DISCUSSION TITLE
looking for coders

### DISCUSSION THREAD

**Discussion by AndyMin2020** [community/community#28247] [Other Feature Feedback, Questions, & Ideas]
Hello,

Looking for fellow coders or people who are interested in like minded projects.
Would like to make a project to do with python or Java. Want to improve my coding skills and teamworking skills.

Cheers.

**Comment by levintech** [comment]
Hi, I am familiar with Python. I'd like to discuss more details with you

**Reply by AndyMin2020** [reply]
You got discord?

**Reply by levintech** [reply]
Levin#4228

**Reply by Runner3131** [reply]
Hello,
I also know Python and learn Kotlin, but I don't know Java but am willing to learn.
I already have a Discord server up and running. Called Student Cloud Computing projects. So be a place for people from my school to get together, code and make cloud projects together. It didn't really work out yet, most of them don't know any coding or their learning, but I got like 6 members, but y'all can join and we can start thinking of a project to do. Maybe make some Discord bots out of Python or something. Here is the link 

https://discord.gg/sm5CQaBptV

And my user name is XxFreeLancerxX, and I need to assign some people roles in this discord. Anyone that can code and up for doing some projects in Python is good enough to be assigned a role if you want.
And it's a group for all students, not just for my school or anything. We have people who started out and were interested in 16 different colleges. That's when we started on Yammer, Microsoft teams. Microsoft Office connects all 16 colleges. That's how I got that many people, but when I started trying to get them set up on Github, and set them up on learning platforms, they started disappearing
But yeah, I would love to do some projects with y'all. I already have the server set up to do project ideas and brainstorming. So just head over, and give a shout.

https://discord.gg/sm5CQaBptV

I can't wait. That;s why I started this group to have someone to do projects with and learn.

**Comment by Runner3131** [comment]
Hello,
I also know Python and learn Kotlin, but I don't know Java but am willing to learn.
I already have a Discord server up and running. Called Student Cloud Computing projects. So be a place for people from my school to get together, code and make cloud projects together. It didn't really work out yet, most of them don't know any coding or their learning, but I got like 6 members, but y'all can join and we can start thinking of a project to do. Maybe make some Discord bots out of Python or something. Here is the link 

https://discord.gg/sm5CQaBptV

And my user name is XxFreeLancerxX, and I need to assign some people roles in this discord. Anyone that can code and up for doing some projects in Python is good enough to be assigned a role if you want.
And it's a group for all students, not just for my school or anything. We have people who started out and were interested in 16 different colleges. That's when we started on Yammer, Microsoft teams. Microsoft Office connects all 16 colleges. That's how I got that many people, but when I started trying to get them set up on Github, and set them up on learning platforms, they started disappearing
But yeah, I would love to do some projects with y'all. I already have the server set up to do project ideas and brainstorming. So just head over, and give a shout.

https://discord.gg/sm5CQaBptV

THis is the Github Hackaton themed server.
I can't wait. That;s why I started this group to have someone to do projects with and learn.
